### PR TITLE
[12.0] [MIG] account_due_list_aging_comment

### PR DESCRIPTION
**Pull request by cor9003**
Migration module account_due_list_aging_comment to 12.0

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why creating another one if #225 is there?

**Comment by cor9003** [comment]
@pedrobaeza The PR #225 must be closed. This PR is the right PR for the migration. In this PR I didn´t follow the right standars  detailed in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0.
I' ve prefered create a PR form scratch in order to learn and do the rigth procedures for this and for future migration. Sorry for the inconveniences caused

**Comment by pedrobaeza** [comment]
OK, done

**Comment by cor9003** [comment]
@mreficent The PR is ready and I appreciate a lot your support in Github. Thanks

File: account_due_list_aging_comment/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2015-2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    'name': "Payments Due list aging comment",
```
Comment (MiquelRForgeFlow) [inline account_due_list_aging_comment/__manifest__.py]: Account Due List Aging Comment

File: account_due_list_aging_comment/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+   :local:
+
+
+
+Usage
+=====
+
+To use this module, you need to go to:
+
+Accounting -> Adviser -> Payments and due list
```
Comment (AaronHForgeFlow) [inline account_due_list_aging_comment/README.rst]: Invoicing -> Accounting -> Payments and due list

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional review LGTM

File: account_due_list_aging_comment/README.rst (hunk 2 above)
Comment (cor9003) [reply account_due_list_aging_comment/README.rst]: Done and also is squash in only one commit. Thanks for your revision

File: account_due_list_aging_comment/__manifest__.py (hunk 1 above)
Comment (cor9003) [reply account_due_list_aging_comment/__manifest__.py]: Done and also is squash in only one commit. Thanks for your revision

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**